Frontend/ui-customer/src/Components/DriverInfoCard/Common/View.purs

```diff
@@ -101,7 +102,7 @@ driverDetailsView config uid nid =
           , gravity LEFT
           ] <> FontStyle.body27 TypoGraphy
         , textView (
-          [ text $ spaceSeparatedPascalCase $ config.vehicleColor <> " " <> config.vehicleModel
+          [ text $ spaceSeparatedPascalCase $ config.vehicleColor <> " " <> if config.vehicleModel == "Unkown" then HU.getVariantRideType config.vehicleVariant else config.vehicleModel
```

Comment:
@slow-codex can we correct this typo error.

Reply:
@MercyQueen This is not a typo, vehicle table stores it as `Unkown` [here](https://github.com/nammayatri/nammayatri/blob/5a29e6bfde9f821146ea815ce9d3f1595d37d860/Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverOnboarding.hs#L268).